lib.py

```diff
@@ -2,8 +2,11 @@
 import numpy as np
 import matplotlib.pyplot as plt
 from tkinter import *
-import tkFileDialog
+
+try:
+    import tkFileDialog
+except:
+    from tkinter import filedialog as tkFileDialog
```

Comment:
Be more specific about the catch of an exception.Because it might delay the execution due to unnecessary catch.